makefiles/tools/riotgen.inc.mk

```diff
@@ -0,0 +1,17 @@
+.PHONY: riotgen-installed bootstrap-%
```

Comment:
IMO "bootstrap" is misleading

Reply:
What would you propose instead ? `init-%`, `generate-%` ?

Reply:
maybe generate-%? or start-%?

Reply:
2 `generate` propositions, let's go for this one